deploy/charts/trust-manager/README.md

```diff
@@ -25,6 +25,7 @@ Kubernetes: `>= 1.25.0`
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | affinity | object | `{}` | Kubernetes Affinty; see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#affinity-v1-core |
+| app.hostNetwork | bool | `false` | Specifies if the app should be started in hostNetwork mode. Required for use in some managed kubernetes clusters (such as AWS EKS) with custom CNI. |
```

Comment:
I think this should be a webhook option (so `app.webhook.hostNetwork`)

Reply:
Also, I think we want to set `dnsPolicy: ClusterFirstWithHostNet` when hostNetwork is enabled (based on what we have in cert-manager): https://github.com/cert-manager/cert-manager/blob/ef3b66aebafa8467ecceb2e8567b4d62b400b2bb/deploy/charts/cert-manager/templates/webhook-deployment.yaml#L58-L60.